.coveragerc

```diff
@@ -1,3 +1,6 @@
+[run]
+branch = True
```

Comment:
I'm pretty sure this was for `py.test`, which doesn't matter for us, we already use `nose-cover` and the correct coverage flags